.github/workflows/release-drafter.yml

```
@@ -0,0 +1,19 @@
+name: Release Drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - main
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
```

```suggestion
      # Drafts your next Release notes as Pull Requests are merged into "main"
```

Tiniest of nits, but master is mentioned here. Might want to change that to "main" should this file stick around.